.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
Do we need all of the 2015 preset? Babel tends to add a lot of bloat with their polyfills. Most es2015 are supported by major browsers theses days. Really all we need support for are ES6 modules, class properties, and the regenerator runtime for async/await. Checkout these

[modules](https://www.npmjs.com/package/babel-plugin-transform-es2015-modules-commonjs)
[class properties](https://babeljs.io/docs/plugins/transform-class-properties/)
[regen](https://babeljs.io/docs/plugins/transform-regenerator/)